packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -195,7 +195,7 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
           ) : undefined
         }
         showLabel={!(shouldFloatLocalVideo && isNarrow)}
-        displayName={localParticipant?.displayName}
+        displayName={isNarrow ? '' : 'You'}
```

Comment:
can we make sure the "You" is using a localized value?

Reply:
Ah, thanks for pointing that out.

Reply:
Changing just display name here to `You` will cause the participant initials to be a `Y` instead of their initials:
![image](https://user-images.githubusercontent.com/2684369/141527370-8b476059-28dd-4365-82b9-073c5193c71d.png)

I think we need to separate the logic of display name and avatar initials inside the video tile

Reply:
Haha yes, my memory is failing me.